libs/langchain/tests/mock_servers/robot/server.py

```diff
@@ -201,5 +201,6 @@ def custom_openapi() -> dict[str, Any]:
 # This lets us prevent the "servers" configuration from being overwritten in
 # the auto-generated OpenAPI schema
 app.openapi = custom_openapi
+
 if __name__ == "__main__":
-    uvicorn.run(app, host="0.0.0.0", port=PORT)
+    uvicorn.run(app, host="0.0.0.0", port=PORT)  # noqa: S104
```

Comment:
Change 0.0.0.0 to 127.0.0.1 / localhost so requests on the local network don't make it through (even if it's testing).

Also this file is probably not used by anything and we could delete it in a bit